bin/install-mediawiki.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
It would be great to one day do this through Docker instead of a wget and a script! :)

Reply:
Or the debian package even.